ajax/libs/react-uwp/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "react-uwp",
+  "description": "UWP Design & Fluent Design UiKit by React",
+  "filename": "css/segoe-mdl2-assets.css",
+  "keywords": [
+    "uwp",
+    "microsoft-uwp",
+    "fluent",
+    "microsoft-fluent",
+    "uwp-design",
+    "fluent-design",
+    "react",
+    "typescript",
+    "javascript",
+    "react-components"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/myxvisual/react-uwp.git"
+  },
+  "author": "myxvisual",
+  "license": "MIT",
+  "homepage": "https://www.react-uwp.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/myxvisual/react-uwp-bower.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jpg|jpeg)"
```

Comment:
Maybe just `!(LICENSE)` can be much more easier and future compatible? cc @cdnjs/intern3 @cdnjs/library-reviewer